docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-tensorflow.rst

```diff
@@ -0,0 +1,131 @@
+Quantization-aware Training (QAT) with TensorFlow
+===================================================
+
+Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with TensorFlow:
+
+.. note::
+   Currently, NNCF for TensorFlow supports optimization of the models created using Keras
+   `Sequential API <https://www.tensorflow.org/guide/keras/sequential_model>`__ or
+   `Functional API <https://www.tensorflow.org/guide/keras/functional>`__.
+
+1. Import NNCF API
+########################
+
+In this step, you add NNCF-related imports in the beginning of the training script:
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [imports]
+
+2. Create NNCF configuration
+####################################
+
+Here, you should define NNCF configuration which consists of model-related parameters (``"input_info"`` section) and parameters
+of optimization methods (``"compression"`` section). For faster convergence, it is also recommended to register a dataset object
+specific to the DL framework. It will be used at the model creation step to initialize quantization parameters.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [nncf_congig]
+
+
+3. Apply optimization methods
+####################################
+
+In the next step, you need to wrap the original model object with the ``create_compressed_model()`` API using the configuration
+defined in the previous step. This method returns a so-called compression controller and a wrapped model that can be used the
+same way as the original model. It is worth noting that optimization methods are applied at this step so that the model
+undergoes a set of corresponding transformations and can contain additional operations required for the optimization. In
+the case of QAT, the compression controller object is used for model export and, optionally, in distributed training as it
+will be shown below.
```

Comment:
```suggestion
3. Apply Optimization Methods
####################################

Wrap the original model object with the ``create_compressed_model()`` API using the configuration
defined in the previous step. This method returns a so-called compression controller and a wrapped model that can be used the
same way as the original model. Optimization methods are applied at this step, so that the model
undergoes a set of corresponding transformations and contains additional operations required for optimization. In case of QAT, the compression controller object is used for model export and, optionally, in distributed training as demonstrated below.
```